src/smsCreditCalc.ts

```diff
@@ -0,0 +1,27 @@
+const utf8Characters = new RegExp(
+  "^[A-Za-z0-9 \\r\\n@£$¥èéùìòÇØøÅå\u0394_\u03A6\u0393\u039B\u03A9\u03A0\u03A8\u03A3\u0398\u039EÆæßÉ!\"#$%&'()*+,\\-./:;<=>?¡ÄÖÑÜ§¿äöñüà^{}\\\\\\[~\\]|\u20AC]*$"
+);
+
+/**
+ * Check if the passed text contains only utf8 characters
+ */
+export const isAllUtf8 = (text: string): boolean => utf8Characters.test(text);
+
+/**
+ * Estimate the number of credits used for the given message content
+ * @param message SMS content
+ * @return Credits estimation as an integer
+ */
+export const creditEstimation = (
```

Comment:
Does it have to be called `creditEstimation`? It is a noun, rather than a verb (for a function name).

Reply:
Poor Suchandra, means re-publishing, (technically a major version bump :P) and getting it updated in Gentu/Website :D